qiita_db/base.py

```diff
@@ -130,11 +130,12 @@ def _check_id(self, id_):
         """
         self._check_subclass()
 
-        conn_handler = SQLConnectionHandler()
-
-        return conn_handler.execute_fetchone(
-            "SELECT EXISTS(SELECT * FROM qiita.{0} WHERE "
-            "{0}_id=%s)".format(self._table), (id_, ))[0]
+        with TRN:
+            sql = """SELECT EXISTS(
+                        SELECT * FROM qiita.{0}
+                        WHERE {0}_id=%s)""".format(self._table)
+            TRN.add(sql, [id_])
```

Comment:
I am under the impression that these had to be tuples and that lists were interpreted in a very different way.